request.go

```diff
@@ -23,16 +36,8 @@ type Request struct {
 	// request.
 	Addr string
 
-	// Cmd is the Redis command that's being sent with this request.
-	Cmd string
-
-	// Args is the list of arguments for the request's command. This field
-	// may be nil for client requests if there are no arguments to send with
-	// the request.
-	//
-	// For server request, Args is never nil, even if there are no values in
-	// the argument list.
-	Args Args
+	// Cmds is the list of transaction's commands.
```

Comment:
`Cmds is the list of transaction's commands.` => `Cmds is the list of commands that are part of the transaction.`